examples/test_spectrum_model.py

```diff
@@ -1,47 +1,79 @@
 """Spectrum model API for Gammapy.
 """
+import numpy as np
 from astropy.units import Quantity as Q
 from gammapy.spectrum.models import (
     SpectralModel, PowerLaw, ExponentialCutoffPowerLaw
 )
+from gammapy.irf import (
+    EffectiveAreaTable,
+    EnergyDispersion
+)
+from gammapy.utils.energy import EnergyBounds
+from gammapy.spectrum import calculate_predicted_counts
+
+
+# TODO clean up and add to gammapy.irf
```

Comment:
Why not move it to an `EnergyDispersion` classmethod now?
It's not a lot of work and then it's directly in the right place and easier to find and use.